core/cap-TBD.md

```diff
@@ -0,0 +1,193 @@
+## Preamble
+
+```
+CAP: To Be Assigned
+Title: Soroban refined VM instantiation cost model
+Working Group:
+    Owner: Graydon Hoare <@graydon>
+    Authors: Graydon Hoare <@graydon>
+    Consulted: Jay Geng <@jayz22>, Dmytro Kozhevin <@dmkozh>
+Status: Draft
+Created: 2024-03-11
+Discussion: https://github.com/stellar/stellar-protocol/discussions
```

Comment:
It'd be helpful to open a discussion there, and link to the specific discussion.

Reply:
```suggestion
Discussion: https://github.com/stellar/stellar-protocol/discussions/1460
```